.vuepress/config.js

```diff
@@ -271,21 +271,6 @@ module.exports = {
             },
           ],
         },
-        {
-          title: '🧪 Experimental',
-          collapsable: false,
-          path: '/learn/experimental/overview.html',
-          children: [
-            {
-              title: 'Overview',
-              path: '/learn/experimental/overview',
-            },
```

Comment:
It will be a good idea if we have a new experimental feature planned in the near future. If not, then I don't think we should keep it. We can always add it back later